app/models/v2/tailoring.rb

```diff
@@ -11,6 +11,8 @@ class Tailoring < ApplicationRecord
     belongs_to :profile, class_name: 'V2::Profile'
     has_one :security_guide, through: :profile, class_name: 'V2::SecurityGuide'
     has_one :account, through: :policy, class_name: 'V2::Account'
+    has_many :tailoring_rules, class_name: 'V2::TailoringRule', dependent: :destroy
```

Comment:
Do we need to use `dependent: :destroy` here? 
The callbacks in the case of a many-to-many helper table are not needed afaik. We're only deleting associations after all.
Also, `dependent: :delete_all` is supposed to be more performant being only one query, this could be faster in case of a tailoring with many rules.

Please correct me if I'm wrong.

Reply:
We should NEVER use `delete_all` as it does not run validations 🙈 the performance increase is exactly because of validations. The `dependent` is only added to make rubocop happy, but if we actually drop a tailoring, it should do destroy for sure.

Reply:
Thanks for clarifying.

Reply:
We might want to change this later, but let's keep it consistent